src/pages/api/auth/login.ts

```diff
@@ -16,8 +16,10 @@ import {
 import { getUserByUsername } from 'queries';
 import * as yup from 'yup';
 import { ROLES } from 'lib/constants';
+import { getIpAddress } from 'lib/detect';
 
 const log = debug('umami:auth');
+const logFailed = debug('umami:auth:failed');
```

Comment:
You don't need to start another log object. You can use the existing one and simply include the IP.

Reply:
Yes, but the reason I did not modified the existing log line is that I thought that it would not be good to log the whole user object (if the user exists) including the password field. If you still want it that way, I can change it.